README.md

```diff
@@ -444,7 +486,8 @@ You can find the documentation [here](docs/Readme.md).
 
 ### Cleanup
 
-Execute the following commands if you wish to clean up the Kubernetes cluster by removing all the applied artifacts and configurations related to API operator and API portal.
+Execute the following commands if you wish to clean up the Kubernetes cluster by removing all the applied artifacts
+and configurations related to API operator and API portal.
 
 ```sh
```

Comment:
We need to update the APICtl installation pointing to product-apim-tooling and update wso2am operator part. THis should point to existing K8s artifacts.